### ISSUE TITLE
Cannot see Mailings>>Mailchimp Settings in CiviCRM Menu

### ISSUE THREAD

**Issue by mfuggle** [veda-consulting/uk.co.vedaconsulting.mailchimp#266]
I recently upgraded to the Mailchimp extension V 2.0 and disabled it when it caused issues in the Manage Groups screen (addition column displaced other columns). I have now removed the code that displays the new column and enabled the Mailchimp extension. However I am unable to find the Mailings>>Mailchimp Settings in any of the menus.
It must be something simple but I cannot see it.

**Comment by PalanteJon** [comment]
I had this same problem on a test server running Ubuntu 16.04LTS - I tried uninstalling and reinstalling and ran into issue #269.  Since neither of these behaviors are happening on my live site, I'm guessing they're related - perhaps by my running MySQL 5.7 on the test server.  When I fixed issue #269, I was able to reinstall and my settings were back.  So try uninstall/reinstall, and if you run into issue #269 a patch is available.
